src/Writer/XLSX/Options/SheetProtection.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenSpout\Writer\XLSX\Options;
+
+final class SheetProtection
```

Comment:
```suggestion
final readonly class SheetProtection
```
1. Please mark the class as `readonly`
2. Make all the properties `public` so we can get rid of the setters/getters